Build fails against versions of Node bundled with npm 3

Travis is building (successfully) against Node 0.10, which is bundled with npm 1.

```
$ nvm use 0.10
Now using node v0.10.41 (npm v1.4.29)
```

Node 0.12 and 4 are bundled with npm 2, which warns about a change to the way that peer dependencies will be (are) handled in npm 3. The build succeeds against both:

```
$ nvm use 0.12
Now using node v0.12.9 (npm v2.14.9)
$ npm install && npm test
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-jshint will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-watch will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
…
Done, without errors.
```

```
$ nvm use 4
Now using node v4.2.6 (npm v2.14.12)
$ npm install && npm test
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-jshint will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-contrib-watch will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
…
Done, without errors.
```

Building against Node 5, bundled with npm 3, fails:

```
$ nvm use 5
Now using node v5.5.0 (npm v3.3.12)
$ npm install && npm test
…
npm WARN EPEERINVALID grunt-contrib-watch@0.6.1 requires a peer of grunt@~0.4.0 but none was installed.
npm WARN EPEERINVALID grunt-contrib-jshint@0.10.0 requires a peer of grunt@~0.4.0 but none was installed.
…
Fatal error: Unable to find local grunt.
…
npm ERR! Test failed.  See above for more details.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,3 +1,6 @@
 language: node_js
 node_js:
+  - '5'
+  - '4'
+  - '0.12'
   - '0.10'

--- a/package.json
+++ b/package.json
@@ -23,6 +23,7 @@
     "string-argv": "^0.0.1"
   },
   "devDependencies": {
+    "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-contrib-jshint": "^0.10.0",
     "grunt-contrib-watch": "^0.6.1",

